src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -0,0 +1,158 @@
+/*
+ *  Copyright 2021 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+const config = require('../../lib/config');
+const commons = require('../../lib/commons');
+const selectors = require('../../lib/util/forms.selectors.js');
+
+describe('AEM Forms Reference Artifacts', () => {
```

Comment:
Can we have meaningful test names ?

Reply:
@Meet001 , change it to "AEM Forms OOTB Content Tests", This is  top level test suite name. I will make these changes in my PR as well.

Reply:
Done